index.html

```diff
@@ -247,7 +247,7 @@ <h1>I'm Zermarden <br>
               </div>
             </div>
           </div>
-        </section>
+        </div>
         <section class="about-myself">
           <div class="top">
             <div class="primary-text-5">
```

Comment:
- [ ] There should be only one `h1` in an HTML file because of semantics. I kindly suggest that you make only your main title of "I’m Zermarden Glad to see you!" an `h1` and kindly make use of headers from `h2` down to `h6` for any other headers in your HTML file.

As I'm unable to use inline comments in some of your code. Please kindly:

- [ ] Make your text of LET'S CONNECT a header by giving it a `h2` to `h6`